forest/constants.py

```diff
@@ -26,6 +26,7 @@ class Frequency(Enum):
     THREE_HOURLY = 3
     SIX_HOURLY = 6
     TWELVE_HOURLY = 12
+    MINUTELY = 1/60
```

Comment:
Now that the smallest unit is a minute, would it make more sense to base all values on that and keep as `int` (`MINUTELY = 1`, `HOURLY = 60`, etc) or convert all values to `float`? One benefit of `int` is that the values are precisely represented in binary but there could be other considerations for other Forest trees?

Also, in the context of #166 we should probably change `HOURLY_AND_DAILY` to something like `0` or `-1` for consistency. What do you think?

Reply:
I agree with this, but since it's affecting all the other trees as well maybe it's better to have a separate PR where we focus on the `Frequency` class and we make everything an `int`

Reply:
OK. I did a quick search (https://github.com/search?q=repo%3Aonnela-lab%2Fforest%20.value&type=code) and the only other places where the enum value is used are:
* https://github.com/onnela-lab/forest/blob/55c9fd9fbc1b60cf5f15ed195af2cd3fce1de831/forest/jasmine/traj2stats.py#L1155
* https://github.com/onnela-lab/forest/blob/55c9fd9fbc1b60cf5f15ed195af2cd3fce1de831/forest/willow/log_stats.py#L340

If this is correct and since this PR is about adding the new minute-level value I think we should address it here.